plugins/ui-kit-candidates/HeaderBar.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { coreColors } from "@dcos/ui-kit/dist/packages/shared/styles/color";
+
+const headerBarStyles: object = {
+  backgroundColor: coreColors().purpleDarken4,
+  height: "32px",
+  width: "100%",
+  flexShrink: 1,
+  flexGrow: 0,
+  flexBasis: "auto",
+  zIndex: 999
```

Comment:
Would it make sense to put this into a constant, so we have named levels?

Reply:
We could introduce something like that in UI Kit, yeah. (not in scope for this PR I'd say)

Reply:
Hmm, having UI Kit here is a problem. Ok nevermind

Reply:
I dont think it is a problem actually, UI Kit can efine levels for certain types of components.